plugins/module_utils/gitlab.py

```diff
@@ -119,8 +119,18 @@ def gitlab_authentication(module, min_version=None):
             resp_data = resp.json()
             gitlab_oauth_token = resp_data["access_token"]
 
-        gitlab_instance = gitlab.Gitlab(url=gitlab_url, ssl_verify=verify, private_token=gitlab_token,
-                                        oauth_token=gitlab_oauth_token, job_token=gitlab_job_token, api_version=4)
+        kwargs = {
+            'url': gitlab_url,
+            'ssl_verify': verify,
+            'private_token': gitlab_token,
+            'oauth_token': gitlab_oauth_token,
+            'job_token': gitlab_job_token,
+            'api_version': 4,
+        }
+        if gitlab.__version__ >= LooseVersion('3.8.0'):
```

Comment:
While this might work (`LooseVersion` does some magic), it's better to explicitly convert both version strings to `LooseVersion`:
```suggestion
        if LooseVersion(gitlab.__version__) >= LooseVersion('3.8.0'):
```